tests/spread/charms/centos-7/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: pack default charm for centos 7
+manual: true
```

Comment:
why is this manual?

Reply:
I see the tests under this category are all manual. Should it be `false`?

Reply:
move it to smoketests, the "charms" suite was meant for actual production charms to be tested

@lengau perhaps move bundle from there too ^ ?